products/eap/get-started.adoc

```diff
@@ -20,11 +20,11 @@ Prerequisites &#38; Required Dependencies Checklist
 == Step1 Content
 1.  Install the Java SE Development Kit (JDK), we recommend OpenJDK or
 Oracle JDK.
-2.  Download link:http://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-standalone_jar.jar?tp=eap[JBoss Developer Studio 8.1.0.GA with EAP]
+2.  Download link:https://developers.redhat.com/download-manager/content/origin/files/sha256/1e/1efef4d2b109efd9e8136d02067a8d62e00621e5b2b1530811bf52b5aa2fca6f/jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar?tp=eap[JBoss Developer Studio 9.0.0.GA with EAP]
```

Comment:
Ah yes, i'd not noticed that. We need a bigger fix here to overcome the problem. I suggest we go ahead with this as-as, and we'll fix the production download manager URL as part of: https://issues.jboss.org/browse/RHD-1348